sp800-63b/sec5_authenticators.md

```diff
@@ -399,4 +399,8 @@ The latter class of verifier impersonation resistant protocols relies on access
 
 In contrast, authenticators that involve the manual entry of an authenticator output, such as out of band and one-time password authenticators, SHALL NOT be considered verifier impersonation resistant because they assume the vigilance of the claimant to determine that they are communicating with the intended verifier.
 
+#### <a name="csp-verifier"></a>5.2.6. Verifier-CSP commmunications
 
+In situations where the verifier and CSP are separate entities, communications between the verifier and CSP SHALL occur through a mutually-authenticated secure channel (such as a client-authenticated TLS connection) using approved cryptography.
+
+Verification of certificates using Open Certificate Status Protocol (OCSP) [[RFC 6960]](#RFC6960) SHOULD occur through an authenticated protected channel. OCSP responses SHALL be signed as specified in the OCSP specification. The verifier SHALL check responses for validity and SHALL NOT depend upon any invalid responses received. 
```

Comment:
This is way too prescriptive for my liking.  OCSP is just checking for status of a credential (or in this case, potentially of the servers for of the CSP/Verifier).   I would prefer something like - CSP-Verifier SHALL be mutually authenticated, server credentials SHALL be PK based.  ...  What do you think?